zulipterminal/core.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, config_file: str, maximum_footlinks: int,
         self.color_depth = color_depth
         self.in_explore_mode = in_explore_mode
         self.autohide = autohide
-        self.notify_enabled = notify
+        self.notify = notify
```

Comment:
Let's leave the controller attribute descriptive too.

Reply:
(I noticed this since this commit now modifies more files than indicated in the commit)